### DISCUSSION TITLE
增加可嵌入博客的[博主最后在线时间]功能

### DISCUSSION THREAD

**Discussion by DomeenoH** [Qexo/Qexo#94] [Ideas]
类似hpp的签到功能，感觉很有意思，希望qexo能有个类似的api

**Comment by am-abudu** [comment]
哈哈 就是获取登陆时间之类的吗

**Reply by DomeenoH** [reply]
对对，最后一次登陆的时间，或者最后一次push的时间，感觉都可以

**Comment by am-abudu** [comment]
8ea8e4daf4ab8f1795d6203456c6b2f0e24c31d8

**Reply by DomeenoH** [reply]
~~可以写一个引入的教程吗~~  
研究了半天，结果没搞懂应该如何引入网页中
